.buildbot_dockerfile_debian

```diff
@@ -6,4 +6,4 @@ RUN apt-get update && \
 WORKDIR /ci
 RUN chown ${CI_UID}:${CI_UID} .
 COPY --chown=${CI_UID}:${CI_UID} . .
-CMD sh -x .buildbot.sh
+ENTRYPOINT ["sh", "-x", ".buildbot.sh"]
```

Comment:
Is `ENTRYPOINT` preferred?

Reply:
yeah I think that's the general practice for the main entry of the container
From my understanding:
CMD - can be easily overridden 
ENTRYPOINT - cannot be overridden unless explicitly specified by `--entrypoint`

same but different :)

Reply:
so for example, if I have this dockerfile with CMD:
```
FROM debian:bullseye
ARG CI_UID
RUN useradd -m -u ${CI_UID} ci
RUN apt-get update && \
    apt-get -y install build-essential curl git procps python3
WORKDIR /ci
RUN chown ${CI_UID}:${CI_UID} .
COPY --chown=${CI_UID}:${CI_UID} . .
CMD ["sh", "-x", ".buildbot.sh"]
```
I can build and run it as:
```
$  docker run -t test 
```
that will run the `CMD` command

but I can also override it by overriding `CMD`:
```
$ docker run -t test:latest ls -la
```